docs/content/any/guides/roles/_index.md

```diff
@@ -2,52 +2,72 @@
 title: "Build Role-Based Access Control (30 min)"
 weight: 1
 showContentForAnyLanguage: true
+aliases:
+  - /python/getting-started/roles/builtin_roles.html
 ---
 
 # Build Role-Based Access Control (30 min)
 
 When managing access to resources within an application, it can be
 useful to group permissions into **roles**, and assign these roles to
 users. This is known as **Role-Based Access Control (RBAC).** The Oso
-Roles library provides a configuration-based approach to adding
+Roles feature provides a configuration-based approach to adding
 role-based access control to your application.
 
-The roles library includes:
+The roles feature includes:
 
 - **Role configuration** - Declarative configuration for roles and
   permissions for each resource. The roles configuration supports
   multi-tenancy, resource-specific roles, and hierarchical roles. Groups
   and custom roles are coming soon.
-- **Data management** - Manage user role assignments in your database,
-  linking with your resource data.
 - **Enforcement** - Enforce authorization consistently throughout your
   application routing and data access layers.
-- **End-user configuration** - Expose authorization configuration to
-  end users using Oso's role data API.
 - **Last-mile customizations** - Extend authorization logic for each resource
   by writing custom policies using Polar, Oso's declarative policy
   language.
 
 {{% ifLang "python" %}}
+## SQLAlchemy
+
+If you are using SQLAlchemy for managing your application data, you
+can use the `sqlalchemy-oso` framework integration to additionally handle:
+
+- **Data management** - Manage user role assignments in your database,
+  linking with your resource data.
+- **End-user configuration** - Expose authorization configuration to
+  end users using Oso's role data API.
+
+[Check out the library documentation for SQLAlchemy](./sqlalchemy/getting-started)
 
 ## Get started
 
-{{< tryInFramework >}}
+Continue on to the [getting started guide](./getting-started) to see how to
+add Oso roles to a Python application.
+
+{{% /ifLang %}}
+
+{{% ifLang "node" %}}
+
+## Get started
 
 {{% /ifLang %}}
 
 {{< ifLangExists >}}
+{{% ifLang not="node" %}}
 {{% ifLang not="python" %}}
 
 ## Get started
```

Comment:
is it just me or is this going to render a pair of `Get started` headings if Node.js is the selected language?